PocketKit/Sources/PocketKit/Premium/PremiumUpgradeView.swift

```diff
@@ -5,10 +5,14 @@ struct PremiumUpgradeView: View {
     @Environment(\.dismiss)
     private var dismiss
 
+    @ObservedObject
+    var viewModel: PremiumUpgradeViewModel
+
     var body: some View {
         VStack(spacing: 0) {
             dismissButton
             upgradeView
+                .modifier(offlineAlert(isPresented: true))
```

Comment:
why a custom modifier vs using the `.alert` directly. also do we want `isOffline` here instead of true?